### PR TITLE
change default templates to debian-9 and fedora-26

### PR DESCRIPTION
**Pull request by awokd**
https://github.com/QubesOS/qubes-issues/issues/3426

### CODE REVIEW

File: conf/comps-qubes.xml [hunk 1]
```diff
@@ -1171,12 +1171,12 @@
...
   <group>
-    <id>debian-8</id>
-    <name>Debian 8 (jessie) template</name>
+    <id>debian</id>
+    <name>Debian 9 (stretch) template</name>
     <default>true</default>
     <uservisible>true</uservisible>
     <packagelist>
-      <packagereq>qubes-template-debian-8</packagereq>
+      <packagereq>qubes-template-stretch</packagereq>
```
Comment (marmarek) [inline conf/comps-qubes.xml]: Package name should be qubes-template-debian-9. See how it is done for debian-8 in builder.conf (TEMPLATE_ALIAS).

**Review by marmarek — CHANGES_REQUESTED** [review]

File: conf/comps-qubes.xml (hunk 1 above)
Comment (awokd) [reply conf/comps-qubes.xml]: OK, I think the commit below and the one I just added to qubes-builder should address it. I want to do a full build to make sure but those are taking me ~6 hours so I'm going to incorporate a couple other tweaks first...

Comment (awokd) [reply conf/comps-qubes.xml]: Should I cancel those PRs and resubmit once I've verified?

Comment (marmarek) [reply conf/comps-qubes.xml]: No, keep this one open and let me know the result.

**Review by marmarek — APPROVED** [review]

**Comment by marmarek** [comment]
Since last two commits together does not change anything, I've merged only the first one. Thanks @awokd !

**Pull request closed without merging**